rclcpp/include/rclcpp/publisher_base.hpp

```diff
@@ -133,6 +133,12 @@ class PublisherBase : public std::enable_shared_from_this<PublisherBase>
   size_t
   get_subscription_count() const;
 
+  /// Get non local subscription count
```

Comment:
I think this means that `Not in the same context` for the ROS 2 user (precisely for DDS-RTPS, GUID prefix is not the same), that would be nicer to add the description explains?